src/main/java/org/fungover/mmotodo/team/Team.java

```diff
@@ -5,13 +5,15 @@
 import jakarta.validation.constraints.Size;
 import lombok.Getter;
 import lombok.Setter;
-import org.fungover.mmotodo.user.User;
 import org.fungover.mmotodo.task.Task;
+import org.fungover.mmotodo.user.User;
```

Comment:
This change looks like some previous commit didn't order the imports alphabetical which should be the default setting in IntelliJ when reformatting the code.